.github/workflows/app-build-verify.yml

```diff
@@ -254,28 +257,30 @@ jobs:
     # alive.
     #
     # Only runs when target platform and format are explicitly Linux and AppImage.
-    - name: Build AppImage project
+    - name: Build AppImage Project
       if: >
         startsWith(inputs.runner-os, 'ubuntu')
         && contains(fromJSON('["Linux"]'), inputs.target-platform)
         && contains(fromJSON('["AppImage"]'), inputs.target-format)
+      working-directory: ${{ steps.create.outputs.project-path }}
       run: |
-        cd tests/apps/verify-${{ inputs.framework }}
-        briefcase create linux AppImage
+        briefcase create linux AppImage \
+          ${{ steps.output-format.outputs.template-override }} \
```

Comment:
They all need it; I will add them once https://github.com/beeware/briefcase/pull/1542 is merged. If I specify `--config` now, everything will fail so I only did it for AppImage since it always fails without `--config`.